qiskit/circuit/__init__.py

```diff
@@ -345,10 +345,7 @@
 Random Circuits
 ---------------
 
-.. autosummary::
-   :toctree: ../stubs/
-
-   random.random_circuit
+.. autofunction:: qiskit.circuit.random.random_circuit
```

Comment:
This particular one will need an update to the `.. currentmodule` (and restoration afterwards), because it's not available in the root `qiskit.circuit`, but needs to be imported from `qiskit.circuit.random`.  The new `autofunction` directive will cause it to be documented as if it's available in `qiskit.circuit`, which will be confusing.